src/pages/viewer-app/viewer-app.tsx

```diff
@@ -148,6 +152,11 @@ export const ViewerApp = () => {
     setActiveLayers([newActiveLayer]);
     dispatch(setColorsByAttrubute(null));
     dispatch(setDragMode(DragMode.pan));
+    dispatch(setDebugOptions({ loadTiles: true }));
```

Comment:
Looks like loadTiles state should be set to true by default (in the initial state of the redux slice) and we don't to set it here

Reply:
1. defined loadTiles as true in the debug-option-slice.ts
   2. removed dispatch(setDebugOptions({ loadTiles: true })); from debug-app.tsx, viewer-app.tsx, comparison.tsx
   3.  removed unnecessary import